src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1138,9 +1128,11 @@ namespace DataHeatBalance {
         // Members
         std::string Name;
         int ZonePtr = 0;
+        int spaceIndex = 0; // Space index for this mixing instance
         int SchedPtr = 0;
         Real64 DesignLevel = 0.0;
         int FromZone = 0;
+        int fromSpaceIndex = 0; // Source space index for this mixing instance
```

Comment:
I can see fromSpaceIndex is not used and I assume spaceIndex is also not used, since mixing objects are at the zone level.

Reply:
This was originally going to part of this PR, but will be a separate one to support space for Mixing and CrossMixing